components/match2/wikis/dota2/match_group_input_custom_match_page.lua

```diff
@@ -0,0 +1,555 @@
+---
+-- @Liquipedia
+-- wiki=dota2
+-- page=Module:MatchGroup/Input/Custom/MatchPage
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local Logic = require('Module:Logic')
+local Operator = require('Module:Operator')
+
+local CustomMatchGroupInputMatchPage = {}
+
+function CustomMatchGroupInputMatchPage.getMap(mapInput)
+	-- If no matchid is provided, assume this as a normal map
+	if not mapInput or not mapInput.matchid then
+		return mapInput
+	end
+
+	--local map = mw.ext.LeagueOfLegendsDB.getData(mapInput.matchid, Logic.readBool(mapInput.reversed))
```

Comment:
still intended to use the sample data?

Reply:
Yup

Reply:
sry didn't see that on phone
will give it an additional run thrugh in a bit